src/ValueObject/PhpVersionFeature.php

```diff
@@ -798,4 +803,10 @@ final class PhpVersionFeature
      * @var int
      */
     public const DEPRECATE_NULL_ARG_IN_ARRAY_KEY_EXISTS_FUNCTION = PhpVersion::PHP_85;
+    
+    /**
+     * @see https://wiki.php.net/rfc/deprecations_php_8_5#deprecate_passing_string_which_are_not_one_byte_long_to_ord
+     * @var int
+     */
+    public const DEPRECATE_ORD_WITH_MULTIBYTE_STRING = PhpVersion::PHP_85;
```

Comment:
this constant seems from your other PR, please only add commit/change from each branch PR, checkout from main branch first before checkout new branch

Reply:
Sorry.. It is already merged https://github.com/rectorphp/rector-src/blob/main/src/ValueObject/PhpVersionFeature.php